bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -242,7 +259,7 @@ public String getKey() {
 
     @Override
     public void storeKey(@Nullable String key) {
-        if (!getKey().equals(key)) {
+        if (!disposed && !getKey().equals(key)) {
```

Comment:
does this ever happen?

Reply:
This is the code from the PR #7299 So please consider moving your comment in the right PR.

This can happen because the websocket thread is closed asynchronously and so can call the thing handler (ConfigProvider) while the thing handler is already disposed.

Reply:
Removed from this PR